### DISCUSSION TITLE
Luce automatica con Sensore di presenza

### DISCUSSION THREAD

**Discussion by Redgitblue** [indomus/forum#3231] [Nuove idee, proposte, progetti e guide]
Ciao a tutti,
Sono nuovo sul forum e spero di aver creato correttamente questa richiesta.
Ho da poco iniziato ad interessarmi alla domotica e vorrei iniziare un primo progetto in casa mia per iniziare un po' ad impratichirmi e capire funziona per poi espandere le applicazioni.
Pensavo di partire dal rendere intelligente la luce del terrazzo per far sì che si accenda automaticamente quando uno esce e resti accesa finché c'è qualcuno fuori.
Volevo chiedervi una mano per capire se sono sulla strada giusta o se mi manca qualche pezzo.
Per iniziare avevo pensato di prendere:

- Raspberry Pi su cui installare HA
- Switch smart e ho trovato lo Shelly 1L oppure il  SONOFF ZBMINIL2 (ho selezionato questi due perché dove ora c'è l'interruttore della luce c'è solo 1 dei poli)
- sensore di presenza  (es. SONOFF SNZB-06P)
- Gateway zigbee (es.SONOFF Zigbee Dongle-P)

Le domande che ho sono:

1. È corretto che queste sono le cose che mi servono o manca qualcos'altro?
2. Quanta RAM deve avere il raspberry PI? Ho trovato questo starter kit che mi sembra interessante, ma non so quale capacità scegliere Raspberry] e non ho trovato mini-pc più convenienti
3. Che versione di HA mi conviene installare? Dalle guide ho visto che è consigliata la CORE, ma non riesco a capire quanto sia più complesso gestire l'installazione dei dispositivi successivamente, cosa mi consigliate?
4. Meglio lo shelly o il Sonoff come switch? Quali sono le differenze? 
5. Installando HA non dovrei avere difficoltà anche se scegliessi lo switch Shelly e gli altri componenti Sonoff, giusto?

Scusate per le tantissime domande, ma leggendo le guide sul sito non sono riuscito a darmi una risposta. Se potete darmi delle indicazioni vi ringrazio.

**Comment by KC-inDomus** [comment]
1) tendenzialmente sì
2) raspberry pi almeno 4, ma lascia perdere raspberry
3) core. ma se hai difficoltà, home assistant os
4) sonoff meglio costruiti secondo me, ma anche shelly, non è male, si equilvagono, shelly li integri meglio
5) come sopra

raspberry vs mini pc: https://indomus.it/formazione/domotica-su-home-assistant-mi-conviene-un-nuc-un-raspberry-pi-o-altro/

**Reply by Redgitblue** [reply]
Grazie mille per tutte le risposte chiarissime.
Capisco perche mi consigli il mini PC!
Li ho studiato un po' ma fatico a scegliere il modello!
Hai qualche suggerimento? Vorrei spendere meno di 200€.
Da quello che ho trovato i due che mi sembrano più interessanti sono questo Lenovo](https://amzn.to/4ew7g0m) ricondizionato o questo [beelink](https://amzn.to/3XUmuqx). Del Lenovo mi preoccupa che possa essere vecchio e usurato e che possa consumare molto se lasciato sempre acceso. Del beelink non so quanto sia sicuro (ho letto di parecchio malware nei mini PC cinesi). 
Cosa mi consiglieresti?
Grazie ancora

**Reply by KC-inDomus** [reply]
il malware (eventuale) è un non problema, dato che la macchina appena ti arrivi manco l'accendi: la spiani direttamente con linux.
nessuno dei due, valuta [questo](https://amzn.to/3ziANvB) e [questo](https://amzn.to/4dfda4M), bastano e avanzano (specie il secondo)

**Reply by Redgitblue** [reply]
Come maii consigli il nipogi rispetto al beelink, guardandoli mi sembrano molto simili sia per prezzo che per performance. L'unica differenza mi sembra la CPU che nel beelink è la n95 mentre nel nipogi è la n97 (un po' più potente). È per questo o c'è altro che mi sto perdendo?

**Reply by KC-inDomus** [reply]
è più potente e sopratutto consuma meno
